.env.example

```
@@ -76,6 +76,41 @@ ANTHROPIC_API_KEY=sk-ant-...
+# for real users.
+# -----------------------------------------------------------------------------
+
+# [required in prod] Resend API key — re_...
+# Set as a Convex env var (via `npx convex env set AUTH_RESEND_KEY ...`)
+# so backend auth flows can deliver verification and reset codes.
+AUTH_RESEND_KEY=re_...
+
+# [optional] From-address used on every auth email. Must be a verified
+# Resend sender. Defaults to Resend's shared onboarding address, which
+# only works for testing.
+AUTH_EMAIL=First90 <onboarding@resend.dev>
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)